### PR TITLE
KVM 16.1.1 release

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/20424

Releasing only this kvm-operator change https://github.com/giantswarm/kvm-operator/pull/1313

### CODE REVIEW

File: kvm/requests.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 releases:
-    - name: "> 16.1.0"
+    - name: ">= 16.2.0"
```
Comment (tfussell) [inline kvm/requests.yaml:2]: Adjusting requests as we want this release to be as small as possible.

File: kvm/v16.1.1/release.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    version: 2905.2.3
+  - name: etcd
+    version: 3.5.0
+  - name: flannel-operator
+    version: 0.2.0
+  - name: kubernetes
+    version: 1.21.5
+  - name: kvm-operator
+    releaseOperatorDeploy: true
+    version: 3.18.4
```
Comment (tfussell) [inline kvm/v16.1.1/release.yaml:48]: Will release momentarily.

Comment (tfussell) [reply kvm/v16.1.1/release.yaml:48]: Done https://github.com/giantswarm/kvm-operator/releases/tag/v3.18.4

**Review by pipo02mix — APPROVED** [review]
LGTM

**Pull request merged**